api/descriptions/policies/compliance_ci_serverless_put.md

```diff
@@ -0,0 +1,53 @@
+Updates the compliance policy for serverless functions built in your Continuous Integration (CI) pipeline>.
```

Comment:
@KICKdesigns Stray `>` char

Reply:
@KICKdesigns I fixed this with a commit to your branch.

Reply:
@iansk This is odd. I had pushed up a change removing that character yesterday. Perhaps the push didn't go through.